packages/ui/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact ui module, newest changes on the top.
 
+## [unreleased]
+
+### Added
+
+- `ui/ProgressBar` add `separator` for support balance slider
```

Comment:
Should read like the sentence starts with 'Added'.  So, '(addd) `ui/ProgressBar` prop `separator`'  (I think we don't have to specify what it's for exactly, or, if we do, we should mention the prop it works with.

Reply:
I changed the comment